mulesoft/docs-site-ui
# W 10728368 - survey banner

ref: [W-10728368](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00000qTEcjYAG/view)

### Enhancements

- added survey to all connector pages (moved the survey section into article.hbs)
- added `margin:auto` to survey CSS to align it to the center
- I have noticed that in mid screen size, the survey banner doesn't look right, usually it is too long for the section. Therefore, I standardized the CSS for the survey banner for sm/md/lg screen sizes to match what's there for the sm screen size

---

### How to test

- run `gulp preview` and open localhost:5252/. Scroll down to see that survey banner is aligned to the center of the main viewport section. Drag the left/right edges of the browser to decrease width and see that the banner will change to a square-like shape
- go to the [Salesforce connector](http://localhost:5252/connector.html) and see that the survey button is present
- click on the "take the survey" button opens the userzoom page in a new tab
- http://localhost:5252/connectors.html is the only page that does not have the survey. Unsure if it needs to be there, please clarify
